_translations/en.yml

```diff
@@ -676,9 +676,12 @@ en:
     anonymous: "Bitcoin is not anonymous"
     anonymoustxt: "Some effort is required to protect your privacy with Bitcoin. All Bitcoin transactions are stored publicly and permanently on the network, which means anyone can see the balance and transactions of any Bitcoin address. However, the identity of the user behind an address remains unknown until information is revealed during a purchase or in other circumstances. This is one reason why Bitcoin addresses should only be used once. Always remember that it is your responsibility to adopt good practices in order to protect your privacy. <a href=\"#protect-your-privacy#\"><b>Read more about protecting your privacy</b></a>."
     instant: "Unconfirmed transactions aren't secure"
-    instanttxt: "A Bitcoin transaction is usually deployed within a few seconds and begins to be confirmed in the following 10 minutes. During that time, a transaction can be considered authentic but still reversible. Dishonest users could try to cheat, which means there is a risk when accepting unconfirmated transactions. For large amounts like 1000 US$, it makes sense to wait for 6 confirmations or more. Each confirmation <i>exponentially</i> decreases the risk of a reversed transaction."
+    instanttxt: "New transactions start out as unconfirmed. If you trust the person paying you, you don't have to worry about unconfirmed transactions. If you don't trust them, you should wait for the transaction to receive at least one confirmation—this has historically provided a better than 95% guarantee that the payment can't be changed. Because Bitcoin miners sometimes misbehave, this guarantee isn't as strong for lightweight wallets, so you may want to wait for three confirmations unless you use a recent version of <a href=\"#download#\">Bitcoin Core</a>. For high value amounts, waiting for six or more confirmations is strongly recommended no matter what wallet you use."
```

Comment:
@saivann rather than try to pack all that information into a paragraph, how do you feel about using a table?  Here's an example; it's about 50% taller than your 5-line recommendation but it has lots of white space and I think it's much more readable:

![screenshot-btcorg localhost 2015-07-25 15-25-00](https://cloud.githubusercontent.com/assets/61096/8890936/7171c692-32e1-11e5-9fbc-331d2267bb8a.png)

Although I dislike using abbreviations, I used "confs." so that it would look good on even the least-wide mobiles:

![screenshot-btcorg localhost 2015-07-25 15-24-10](https://cloud.githubusercontent.com/assets/61096/8890939/93c752d4-32e1-11e5-9492-7644ac92cdc3.png)

"Emergencies" links to the alerts overview page with its RSS feed.

If you think this is reasonable, I'll turn it into translatable strings and add it to this PR.